File: Python/A1/A1-005.py
# Author: Korawit Suporn
# Date: 23 Apr. 2025
# Editor: Vim

m = int(input())
d = int(input())

# winter

if ((m == 12 and d >= 21) or (m >= 1 and m <= 2) or (m == 3 and d < 21)):
    print("winter")
elif (m == 3 and d >= 21) or (m >= 4 and m <= 5) or (m == 6 and d < 21):
    print("spring")
elif (m == 6 and d >= 21) or (m >= 7 and m <= 8) or (m == 9 and d < 21):
    print("summer")
else:
    print("fall")

File: C.C++/A1/A1-005.cpp
#include <iostream>
using namespace std;

int main()
{
    int m, d;

    // input
    cin>> m>> d;

    // process
    if (m >= 1 && m <= 3)
    {
        if (m == 3 && d >= 21)
        {
            // output
            cout<< "spring";
        } else {
            cout<< "winter";
        }
    }
        
    if (m >= 4 && m <= 6)
    {
        if (m == 6 && d >= 21)
        {
            cout<< "summer";
        } else {
            cout<< "spring";
        }
    }

    if (m >= 7 && m <= 9)
    {
        if (m == 9 && d >= 21)
        {
            cout<< "fall";
        } else {
            cout<< "summer";
        }
    }
    
    if (m >= 10 && m <= 12)
    {
        if (m == 12 && d >= 21)
        {
            cout<< "winter";
        } else {
            cout<< "fall";
        }
    }
}